app/src/main/java/org/systers/mentorship/view/fragments/RequestPagerFragment.kt

```diff
@@ -57,11 +81,4 @@ class RequestPagerFragment: BaseFragment() {
             tvEmptyList.visibility = View.GONE
         }
     }
-
-    private val openRequestDetail: (Relationship) -> Unit =
-            { requestDetail ->
-                val intent = Intent(activity, RequestDetailActivity::class.java)
-                intent.putExtra(Constants.RELATIONSHIP_EXTRA, requestDetail)
-                startActivity(intent)
-            }
 }
```

Comment:
line in the end of the file please. I know this wasn't you, but since we're here ... ;)